ob_v3p0/common_credentials.lines

```diff
@@ -71,7 +71,6 @@ Package Credentials DataModel
         Property description String 0..1                            "A longer description of the evidence."
         Property genre String 0..1                                  "A string that describes the type of evidence. For example, Poetry, Prose, Film."
         Property audience String 0..1                               "A description of the intended audience for a piece of evidence."
-        Property artifacts Artifact 0..*                            "Artifacts that are part of the evidence."
```

Comment:
I forgot to remove this per PR https://github.com/IMSGlobal/ComprehensiveLearnerRecord/issues/291